MockMDS/src/main/java/io/mosip/registration/mdm/dto/DeviceRequest.java

```diff
@@ -17,6 +17,6 @@ public class DeviceRequest {
 	public String deviceStatus;
 	public String firmware;
 	public String certification;
-	public int[] deviceSubId;
+	public String[] deviceSubId;

```

Comment:
is this inline with spec?  In the spec do we have it as array of string

Reply:
Yes Sasi, it is as per spec.